e2e/proxy.ts

```diff
@@ -18,7 +18,8 @@
 import { browser } from 'protractor';
 
 export async function setConfigField(field: string, value: string) {
+
     return browser.executeScript(
-        `window.adf.setConfigField('${field}', '${value}');`
+        "window.adf.setConfigField(`"+field + "`, `" + value + "`);"
```

Comment:
why replacing ES6 feature with ES5 code?

Reply:
yep is a problem when you put in it a string with the ' ' single quote

Reply:
you can try wit for example the search configuration

Reply:
I see, makes sense